### PR TITLE
Ismail branch

### PR DESCRIPTION
**Pull request by IsmailSadiqO**
Added stylings for the entire form
Added logic to create a capsule and capsule contents

### CODE REVIEW

File: TimeCapsuleManagement/forms.py [hunk 1]
```diff
@@ -4,9 +4,25 @@
...
+    )
+
     class Meta:
         model = Capsule
         fields = '__all__'
         widgets = {
-            'unsealing_date': DateTimeInput(attrs={'type': 'datetime-local'}),
+            'name': forms.TextInput(attrs={'class': 'form-input'}),
+            'description': forms.Textarea(attrs={'class': 'form-input', 'rows': 6}),
+            'unsealing_date': forms.DateTimeInput(
```
Comment (Harishcruise) [inline TimeCapsuleManagement/forms.py:25]: Whether the DateTimeInput handle edge cases,
like the user should choose only the date and time which after the current time.

File: TimeCapsuleManagement/views.py [hunk 2]
```diff
@@ -17,9 +20,32 @@ def my_capsules(request):
...
+            for file in files:
+                mime_type, _ = mimetypes.guess_type(file.name)
+                file_type = 'text'  # Default file type
+                if mime_type:
+                    if mime_type.startswith('image/'):
+                        file_type = 'photo'
+                    elif mime_type.startswith('video/'):
+                        file_type = 'video'
+                # Create a CapsuleContent object for each file
+                CapsuleContent.objects.create(
```
Comment (Harishcruise) [inline TimeCapsuleManagement/views.py:35]: Try to call operations functions, or else nvm.

**Review by Harishcruise — CHANGES_REQUESTED** [review]

File: TimeCapsuleManagement/forms.py (hunk 1 above)
Comment (IsmailSadiqO) [reply TimeCapsuleManagement/forms.py:25]: fixed

File: TimeCapsuleManagement/views.py (hunk 2 above)
Comment (IsmailSadiqO) [reply TimeCapsuleManagement/views.py:35]: I will look at how we can do that in a future PR.

**Review by IsmailSadiqO — COMMENTED** [review]
Done

**Review by Harishcruise — APPROVED** [review]

**Pull request merged**